core/src/main/scala/org/http4s/multipart/MultipartParser.scala

```diff
@@ -243,17 +251,9 @@ object MultipartParser {
 
   /** Split a chunk in the case of a partial match:
     *
-    * If it is a chunk that is between a partial match
-    * (middle chunked), the prior partial match is added to
-    * the accumulator, and the current partial match is
-    * considered to carry over.
-    *
-    * If it is a fresh chunk (no carry over partial match),
-    * everything prior to the partial match is added to the accumulator,
-    * and the partial match is considered the carry over.
-    *
-    * Else, if the whole block is a partial match,
-    * add it to the carry over
+    * DO NOT USE. Was made private[http4s] because
+    * Jose messed up hard like 5 patches ago and now it breaks bincompat to
+    * remove.
```

Comment:
If it's marked deprecated, it'll error on actual usage, and we'll know to clean it out.

Reply:
Will mark

Reply:
P.s, this is already cleaned out on master